src/main.cpp

```diff
@@ -33,9 +33,9 @@ long filesize(const char *);
 template <class T> void log_cjson_size(Cjson<T> &, const char *, long);
 
 bool check_arguments(int argc, char *argv[]) {
-    if (argc > 3 || argc == 1) {
+    if (argc != 3) {
```

Comment:
Can you extend this to check that `argv[2]` is one of `-r, -bp, -df`?

Reply:
Done!